packages/just-stack-monorepo/template/rush.json.hbs

```diff
@@ -16,7 +16,7 @@
    * path segment in the "$schema" field for all your Rush config files.  This will ensure
    * correct error-underlining and tab-completion for editors such as VS Code.
    */
-  "rushVersion": "5.5.4",
+  "rushVersion": "5.6.2",
```

Comment:
I think the latest might be 5.6.3? Do we care to go latest?

Reply:
That would be fine. Also just FYI I need to test this more before it goes in.